src/Style/Display/ProgressStyle.ts

```diff
@@ -11,12 +11,17 @@ const spinning = keyframes`
   }
 `;
 
-export const ProgressContent = styled.div`
+export const ProgressContent = styled.div<ProgressContentProps>`
+  size: ${((props: ProgressContentProps) => props.size) as any};  
```

Comment:
Is `size` a legit css attribute? 😮

Reply:
Oh I'm not familiar with typescript but I thought this allowed me to use the `props.size` below.
```
font-size: ${props => `${props.size}em`};
```
Lol sorry was blindly copying and pasting code.